packages/jumpstarter-driver-pyserial/jumpstarter_driver_pyserial/driver.py

```
@@ -1,6 +1,9 @@
 from contextlib import asynccontextmanager
 from dataclasses import dataclass, field
+from typing import Any
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)